tests/cmd/e2e/main.go

```diff
@@ -40,7 +40,10 @@ func main() {
 
 	cli, kubeCli := client.NewCliOrDie()
 
-	context := apimachinery.SetupServerCert(os.Getenv("NAMESPACE"), "webhook-service")
+	context, err := apimachinery.SetupServerCert(os.Getenv("NAMESPACE"), "webhook-service")
```

Comment:
merge this function into `StartValidatingAdmissionWebhookServerOrDie`.

Reply:
done